robot/basestation/static/js/rover.js

```diff
@@ -552,6 +553,54 @@ function gameLoop () {
       }
     }
 
+    // front camera position servo (up/down, servo 1)
+    if ( (millisSince(lastRearPosServoCmd) > POSITION_SERVO_PERIOD) && !$('#servo-val').is(':focus') ) {
+      let returnVals = handlePositionServo (
+        rearTiltPwm, minRearTiltPwm, maxRearTiltPwm,
+        101, 98, // numpad '5' and '2'
+        '#camera-back-tilt-up-btn', '#camera-back-tilt-down-btn'
+      )
+      if (returnVals[0]) {
+        rearTiltPwm = returnVals[1]
+        $('#back-tilt-pwm').text(rearTiltPwm)
+        lastRearPosServoCmd = new Date().getTime()
+        sendRoverCommand('$' + rearTiltPwm.toString())
+      }
+    }
+    // Front camera continuous servo (left/right, servo 2)
+    if ( (millisSince(lastRearContServoCmd) > CONTINUOUS_SERVO_PERIOD) && !$('#servo-val').is(':focus') ){//> CONTINUOUS_SERVO_PERIOD){
+      rearPanPwm = handleContinuousServo (
+        rearPanPwm,
+        97, 99, // numpad '1' and '3'
+        '#camera-back-lpan-btn', '#camera-back-rpan-btn'
+      )
+      // check whether or not to send a new command for the continuous servo
+      if (rearPanPwm == SERVO_STOP && sentRearServoStop) {
+        ;
```

Comment:
Well my intent was to make it more clear logically what I want happening. I'm saying DON'T MOVE if these 2 conditions are true. Otherwise sure you can move

Reply:
I agree with @MartensCedric and I think it's better practice to only create conditions for when something _should_ be done rather than not. I think it was the standard at my first internship. Anyway since ERC is so close and this works I'm not requesting changes.